datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/openmetrics_legacy.yaml

```diff
@@ -179,4 +179,6 @@
       - <TAG_PREFIX:.*>
       - <TAG_SUFFIX$>
 - template: instances/http
+  overrides:
+    request_size.value.example: 10
```

Comment:
Why is this overridden to 10 instead of 16?

Reply:
To keep current behavior https://github.com/DataDog/integrations-core/pull/10183#pullrequestreview-764739626